newIDE/app/src/CommandPalette/CommandsList.js

```diff
@@ -112,7 +112,7 @@ const commandsList: { [CommandName]: CommandMetadata } = {
     area: 'PROJECT',
     displayText: t`Apply changes to the running preview`,
   },
-  OPEN_START_PAGE: { area: 'IDE', displayText: t`Open start page` },
+  OPEN_START_PAGE: { area: 'IDE', displayText: t`Show Home` },
```

Comment:
change key as well?

Reply:
I don't know if it's worth changing all the references to the start page for "home" since it could be seen only as a labelling issue. Start page is still easy to me. But if it bugs you, I can do it